.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "pavittarx.moleculer-snippets"
```

Comment:
This is not maintenance, the last update was five years ago. It shouldn't be suggested as a good extension.